lib/AddMarlin.cmake

```diff
@@ -157,6 +157,8 @@ add_library(
   )
 
 target_compile_features(Marlin PUBLIC cxx_std_14)
-target_include_directories(Marlin PUBLIC Marlin/Marlin/src Marlin/Marlin Marlin)
+target_include_directories(
+  Marlin PUBLIC Marlin/Marlin/src Marlin/Marlin/src/gcode/lcd Marlin/Marlin Marlin
+  )
```

Comment:
is adding "Marlin/Marlin/src/gcode/lcd" really correct?

Reply:
unfortunately this is used for print time etc. in GUI there is similar implementation with hard written path to header file used. here it is more cmake approach. Clean solution using Marlin API does not exist